.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+  - 7
```

Comment:
Since even node versions are the LTS releases: Replace `latest` by `7` and skip the `5` altogether?